app/src/main/res/layout/view_page_header.xml

```diff
@@ -26,6 +26,39 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent" />
 
+        <LinearLayout
+            android:id="@+id/call_to_action_container"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom"
+            android:paddingStart="12dp"
+            android:paddingBottom="12dp"
```

Comment:
Should use `layout_margin` for the container, otherwise, the text inside the container will not have correct gravity.
(You can see the issue of ripple animation when long pressing on the button)

Reply:
Fixed the xml issue. Couldn't reproduce the crash, but I found an issue which could be affecting this as well. So please check if it is fixed now.